fortitude/src/rules/style/snapshots/fortitude__rules__style__tests__unnamed-end-statement_S061.f90.snap

```diff
@@ -12,8 +12,19 @@ snapshot_kind: text
 6 | contains
 7 |   subroutine mysub1()
   |
+  = help: Write as 'end type mytype'.
 
-./resources/test/fixtures/style/S061.f90:9:3: S061 end statement should read 'end subroutine mysub1'
+ℹ Safe fix
+2 2 |   implicit none
+3 3 |   type mytype
+4 4 |     integer :: x
+5   |-  end type                      ! catch this
+  5 |+  end type mytype! catch this
```

Comment:
Hmm, seems the node range includes all this whitespace, that's a little annoying.

We probably want a rule, `missing-whitespace-before-comment`, and that would fix this on a subsequent iteration

Reply:
I was just thinking the same, though I imagine that would fall under the regex rules, and I'm not sure how we would implement fixes for those. Is this fixable at the tree-sitter level?